pandoc-slave/pandoc-slave/pandocrunner.cpp

```diff
@@ -3,103 +3,98 @@
 #include <QStringList>
 #include <QProcess>
 
-
-PandocRunner::PandocRunner(QString &pandocExePath, QObject* parent):
+PandocSlave::PandocRunner::PandocRunner(const QString &pandocExePath, QObject* parent):
     QObject(parent)
   , mContent("")
   , mError("")
```

Comment:
Please, remove `mContent("")` and `mError("")` initialization, as they will be initialize automatically.